tools/flang1/flang1exe/main.c

```diff
@@ -78,41 +78,26 @@ static int debugfunconly = -1;
 static LOGICAL ipa_import_mode = FALSE;
 static char *ipa_export_file = NULL;
 static BIGUINT ipa_import_offset = 0;
-static char *who[] = {"init",     "parser",   "bblock", "vectorize", "optimize",
-                      "schedule", "assemble", "xref",   "unroll"};
+static const char *who[] = {"init",      "parser",   "bblock",
+                            "vectorize", "optimize", "schedule",
+                            "assemble",  "xref",     "unroll"};
 #define _N_WHO (sizeof(who) / sizeof(char *))
 static INT xtimes[_N_WHO];
 static LOGICAL postprocessing = TRUE;
 
 /* Feature names for Fortran front-end */
 #if defined(TARGET_LINUX_X8664)
```

Comment:
I do realize that this is not the objective of this patch, but IMO this place really needs detachment of Linux platform from x86_64 architecture.

Reply:
I will create another PR for this concern.